### ISSUE TITLE
Mark non-shipped/3rdparty apps in occ apps:list

### ISSUE THREAD

**Issue by ghost** [owncloud/core#18193]
Quite often users have a broken install due to some non-shipped/3rdparty app. If the advice is given to disable non-shipped/3rdparty via occ quite often the question "which are non-shipped/3rdparty" arises.

Maybe adding the state of shipped or non-shipped/3rdparty apps in the occ apps:list could be useful here.

**Comment by oparoz** [comment]
I like the idea :+1:

**Comment by karlitschek** [comment]
yes. good idea

**Comment by aptalca** [comment]
To build upon that, following an upgrade, certain apps are disabled. But they do show up in the "not-enabled" list even if they are not compatible at all.

I made the mistake of re-enabling one and I lost access to the web gui completely. I had to ssh in and delete the app folder and only then I could access the gui.

In this case, it was not a 3rd party app, it was the server-side encryption app that is no longer used in 8.1.1. But there was no notice of the fact that it is not compatible with 8.1.1
